spec/models/application_choice_spec.rb

```diff
@@ -352,18 +352,20 @@
           .to change(application_choice, :provider_ids).to(expected_ids)
       end
 
-      it 'updates provider_ids if the course is updated' do
+      it 'updates course_option and current_course_option provider_ids if new courses are provided' do
+        new_course_option = create(:course_option, course: course)
         changed_course = create(:course, :with_accredited_provider)
         changed_course_option = create(:course_option, course: changed_course)
 
         expected_ids = [
+          application_choice.course.provider.id,
```

Comment:
could `this point to original_course_option.course` to be explicit?

Reply:
🆗